tasks/main.yml

```diff
@@ -47,6 +47,14 @@
     - reload systemd
     - restart cockpit
 
+- name: Allow cockpit to own custom port in SELinux policy
+  include_role:
+    name: linux-system-roles.selinux
```

Comment:
@richm: Is it legit to simply use them like that? Upstream they are in separate repos, but downstream they'd always be delivered in the same "linux-system-roles" or "rhel-system-roles" package, right? Or should I move the git clone from the test to here?

Reply:
You should use the following form:
```yaml
include_role:
  name: fedora.linux_system_roles.selinux
```
see https://linux-system-roles.github.io/documentation/role-requirements.html

Reply:
Does that work for RHEL as well? Both our [README](https://github.com/linux-system-roles/cockpit#generate-a-new-certificate) and our current tests use `name: linux-system-roles.*`.

Reply:
> Does that work for RHEL as well? Both our [README](https://github.com/linux-system-roles/cockpit#generate-a-new-certificate) and our current tests use `name: linux-system-roles.*`.

It will.  When we build the RPM for RHEL, we already have to do some rebranding of fedora -> redhat and linux-system-roles -> rhel-system-roles - there will be an additional rebranding step to convert fedora.linux_system_roles -> redhat.rhel_system_roles.

Reply:
The README is problematic - I'm not sure how to convey that you can use one of the 4 different versions of the role: `linux-system-roles.cockpit` or `fedora.linux_system_roles.cockpit` or `rhel-system-roles.cockpit` or `redhat.rhel_system_roles.cockpit` in all of the examples - when we do build the collection and/or downstream the cockpit role, we do update the README.